origin-faucet/README.md

```diff
@@ -8,16 +8,39 @@ Our Origin testnet faucet provides OGN for developers on TestNets. The faucet is
 
 ## Using the faucet in local environment
 
-### Prerequisite
+You can start the faucet using either docker or lerna.
 
-  Use origin-box to start an origin-js container.
+### Docker
 
-      docker-compose up origin-js
+ Use origin-box to start an origin-js container.
 
-### Starting the server
+      docker-compose up origin-js
 
  Start the server in the origin-js container
 
      docker exec -w /app/token origin-js node src/app.js --network_ids=999
 
-  The server should start and you can point your browser to http://localhost:5000 to access the faucet web UI.
+### Lerna
+
+  Make sure to have Postgres installed and running locally.
+  
+  Run migration files to create the DB tables.
+  
+        export DATABASE_URL=postgres://origin:origin@localhost/origin
+        lern run migrate --scope origin-faucet 
+
+  Bootstrap lerna
+  
+        lerna bootstrap
+        
+  Start a local blockchain
+  
+        lerna run start --scope origin-js
+        
+  Start the faucet server
+  
+        cd origin-faucet
+        node src/app.js --network_ids=999
+        
+ ### Access the faucet
+   The server should start and you can point your browser to http://localhost:5000 to access the faucet web UI.
```

Comment:
I would find it helpful if there was another section saying how to create another campaign (running fixtures will prune the database of previous campaigns) and just have a comment somewhere saying that `amount` and `budget` values in FaucetCampaign table are both in Wei.

Reply:
Yes, absolutely. Let me add a script to create a campaign that will also make sure all amounts are in wei. Will also add comment to the table mentioning amounts are in wei.

Reply:
I will handle that as a separate PR to keep things easy to review...